### PR TITLE
Remove duplicates from sorted array using 2 pointers 

### PR DESCRIPTION
**Pull request by PSoni8**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

YES

### Description

Added code of Remove Duplicates from sorted array using 2 pointer in cpp.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
#4759

### CODE REVIEW

File: C-Plus-Plus/other/RemoveDuplicatesFromSortedArray.cpp [hunk 1]
```diff
@@ -0,0 +1,62 @@
+// Given a sorted array,remove the duplicates such that each element appears only once and return the new length using two pointers.
+
+// Function removeDuplicates returns modified size of vector which is passed in this function initially.
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/other/RemoveDuplicatesFromSortedArray.cpp]: Use Multi-Line Comments

File: C-Plus-Plus/other/RemoveDuplicatesFromSortedArray.cpp [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+#include<bits/stdc++.h>
+#include<iostream>
+#include<vector>
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/other/RemoveDuplicatesFromSortedArray.cpp]: Remove the header

File: C-Plus-Plus/other/RemoveDuplicatesFromSortedArray.cpp [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+	    if(nums[p2]==nums[p1]){
+	        p2++;
+	        continue;
+	    }
+	    else{
+	        std::swap(nums[++p1],nums[p2++]);
+	    }
+	}
+nums.erase(nums.begin()+p1+1,nums.end());
+return nums.size();
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/other/RemoveDuplicatesFromSortedArray.cpp]: Give a one tab space

File: C-Plus-Plus/other/RemoveDuplicatesFromSortedArray.cpp [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    int ans = removeDuplicates(arr);
+    cout<<"Answer is"<<ans;
+}
+
+/*Time Complexity: O(n)
+Space Complexity: O(1)
+
+Sample input & output :
+
+sample 1: Input - nums = [1,1,2] ; Output: 2, nums = [1,2]
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/other/RemoveDuplicatesFromSortedArray.cpp]: Remove the redundant line

File: C-Plus-Plus/other/RemoveDuplicatesFromSortedArray.cpp [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+Enter size of array 3
+Enter array 1 1 2
+Answer is 2
+
+sample 2:
+Input - nums = [0,0,1,1,1,2,2,3,3,4] ; output: 5, nums = [0,1,2,3,4]
+Enter size of array 10
+Enter array 0 0 1 1 1 2 2 3 3 4
+Answer is 5
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/other/RemoveDuplicatesFromSortedArray.cpp]: Remove the extra test case

File: C-Plus-Plus/README.md [hunk 6]
```diff
@@ -521,3 +521,4 @@ _add list here_
 - [Activity Selection](other/ActivitySelection.cpp)
 - [Job sheduling with given deadlines](other/job_sheduling_with_deadlines.cpp)
 - [Union and Intersection of arrays](other/Union_and_Intersection_of_arrays.cpp)
+- [Remove Duplicates From Sorted Array using 2 pointes](other/RemoveDuplicatesFromSortedArray.cpp)
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/README.md]: Fix the typo `pointers`

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

**Comment by PSoni8** [comment]
@ankitaggarwal23 all suggested changes are modified . Please Check and Merge PR if all go nice.

**Review by ankitaggarwal23 — COMMENTED** [review]
Don't push more than one code file in a single PR.

**Comment by ankitaggarwal23** [comment]
Closing this PR as of now. Open the new PR if you want this to be reviewed and add the proper files.

**Pull request closed without merging**